gneiss/layouts.py

```diff
@@ -1,3 +1,11 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the Modified BSD License.
```

Comment:
until ete is BSD compatible, it is not appropriate to import ete3 objects into a BSD project...